lib/src/clojure_lsp/kondo.clj

```diff
@@ -194,14 +194,31 @@
        :parallel true
        :copy-configs (settings/get db [:copy-kondo-configs?] true)
        :lint [(string/join (System/getProperty "path.separator") paths)]
-       :config {:output {:analysis {:arglists true
-                                    :locals false
-                                    :keywords true
-                                    :protocol-impls true
-                                    :java-class-definitions true}
-                         :canonical-paths true}}}
+       :config {:output {:canonical-paths true}}}
       (with-additional-config (settings/all db))))
 
+(defn ^:private config-for-internal-paths [paths db custom-lint-fn]
+  (-> (config-for-paths paths db)
+      (assoc :custom-lint-fn custom-lint-fn)
+      (assoc-in [:config :analysis]
+                {:arglists true
+                 :locals false
+                 :keywords true
+                 :protocol-impls true
+                 :java-class-usages true
+                 :java-class-definitions true})))
```

Comment:
The config for a single file includes `:context [:clojure.test :re-frame.core]`. Should that be the same here?

Reply:
these configs include the following features from clj-kondo analysis:
- `:clojure.test`: return more analysis related to test, crucial for test-tree feature, IMO completely not needed for external analysis :)
- `:re-frame.core`: More analysis related to reframe subscriptions/events, not needed for external as well, usually this is something needed for projects that use re-frame for cljs projects

Reply:
Sorry... I should have been clearer. I'm not suggesting we add those to external analysis. What I mean is, when we do full-project analysis and referenced-file analysis (internal, but with many paths) should we be including clojure.test and reframe analysis?

Reply:
Yes, that's weird that didn't cause a issue until now 🤔

Reply:
Weird... well, I added it to the full-project and referenced-file analysis. 🤷